style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: hsla(354, 90%, 47%, 0.504);
+}
```

Comment:
- [ ] AS required fro the project, kindly ensure that your `style.css` file includes one class with styles for your header i.e. firstly, you should add a class in your `<h1>...</h1>` tag, then use it to style it in yous `css` file

Reply:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Hello Microverse</title>
    <link rel="stylesheet" href="style.css">
</head>
<body>
    <h1 class="heading">Hello Microverse!</h1>
</body>
</html>

Reply:
Does html file is right ?

Reply:
.heading {
    color: aqua;
  }